Foundation/NSDecimalNumber.swift

```diff
@@ -51,6 +51,10 @@ public class NSDecimalNumber : NSNumber {
         NSUnimplemented()
     }
     
+    public required convenience init(bytes buffer: UnsafePointer<Void>, objCType: UnsafePointer<Int8>) {
+        NSUnimplemented()
```

Comment:
is calling NSRequiresConcreteImplementation() from here OK? we can't call super.init() because it's a convenience initialiser and also NSNumber is the super class (which does have this initialiser)